packages/app-content-pages/src/shared/components/DropdownNav/DropdownNav.js

```diff
@@ -20,11 +20,12 @@ const StyledButton = styled(Button)`
       `}
   }
 
-  ${props =>
-    props.active &&
-    css`
-      background: ${props.theme.global.colors['neutral-1']};
-    `}
+  &[aria-current='true'] {
```

Comment:
`aria-current='page'` would be more appropriate for a list of navigation links. [Léonie Watson's guide to aria-current](https://tink.uk/using-the-aria-current-attribute/) has an example.

Reply:
Does 'page' apply when navigating to a section you can scroll to? The url does change to link to the H2 such as https://www.zooniverse.org/about/team#minnesota, but it's not a new page.

Reply:
`aria-current=location` might be appropriate in this case.